### PR TITLE
added questionable code for adding users. pls fix

### PR DESCRIPTION
**Pull request by lschwetlick**
Hi Tiziano! 

I made some changes to dd users. Don't you want to merge my work?

### CODE REVIEW

File: auth.py [hunk 1]
```diff
@@ -11,6 +11,11 @@ def authenticate(username, password, pwdb):
     if username in pwdb:
         if password == pwdb[username]:
             status = True
+    else:
+        ans = input(f"Would you like to add a new user with this name {username}? (y/n)")
+        if ans == "y":
+            pwdb = add_user(pwdb, username, password, "pwdb.json")
```
Comment (otizonaizit) [inline auth.py:17]: wel, well. Maybe this should be a variable? What is "pwdb.json" doing here?

File: auth.py [hunk 2]
```diff
@@ -22,14 +27,18 @@ def write_pwdb(pwdb, path):
     with open(path, 'wt') as pwdb_file:
         json.dump(pwdb, pwdb_file)
 
+def add_user(pwdb, username, password, path):
+    pwdb[username] = password
+    write_pwdb(pwdb, 'pwdb.json')
```
Comment (otizonaizit) [inline auth.py:32]: as above

File: auth.py [hunk 3]
```diff
@@ -22,14 +27,18 @@ def write_pwdb(pwdb, path):
...
+
 if __name__ == "__main__":
     username, password = get_credentials()
 
-    #pwdb = {"lisa":"asd"}
-    #write_pwdb('pwdb.json', pwdb)
     pwdb = read_pwdb('pwdb.json')
-    status = authenticate(username, password, pwdb)
-    write_pwdb(pwdb, "pwdb.json")
+    status = authenticate(username, password, pwdb)
```
Comment (otizonaizit) [inline auth.py:40]: there is a trailing white space here

Comment (lschwetlick) [reply auth.py:40]: oh no!

**Comment by lschwetlick** [comment]
Can you merge like this? Its still an improvement!

**Pull request merged**